ct/thelounge.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/misc/build.func)
+# Copyright (c) 2021-2024 tteck
+# Author: kristocopani
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+function header_info {
+clear
+cat <<"EOF"
+  ________            __                               
+ /_  __/ /_  ___     / /   ____  __  ______  ____ ____ 
+  / / / __ \/ _ \   / /   / __ \/ / / / __ \/ __ `/ _ \
+ / / / / / /  __/  / /___/ /_/ / /_/ / / / / /_/ /  __/
+/_/ /_/ /_/\___/  /_____/\____/\__,_/_/ /_/\__, /\___/ 
+                                          /____/       
+ 
+EOF
+}
+header_info
+echo -e "Loading..."
+APP="The Lounge"
+var_disk="4"
+var_cpu="2"
+var_ram="2048"
+var_os="ubuntu"
+var_version="24.04"
+variables
+color
+catch_errors
+
+function default_settings() {
+  CT_TYPE="1"
+  PW=""
+  CT_ID=$NEXTID
+  HN=$NSAPP
+  DISK_SIZE="$var_disk"
+  CORE_COUNT="$var_cpu"
+  RAM_SIZE="$var_ram"
+  BRG="vmbr0"
+  NET="dhcp"
+  GATE=""
+  APT_CACHER=""
+  APT_CACHER_IP=""
+  DISABLEIP6="no"
+  MTU=""
+  SD=""
+  NS=""
+  MAC=""
+  VLAN=""
+  SSH="no"
+  VERB="no"
+  echo_default
+}
+
+function update_script() {
+  header_info
+  check_container_storage
+  check_container_resources
+
+  if [[ ! -f /usr/lib/systemd/system/thelounge.service ]]; then msg_error "No ${APP} Installation Found!"; exit; fi
+  RELEASE=$(curl -s https://api.github.com/repos/thelounge/thelounge-deb/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+  if [[ "v${RELEASE}" != "$(sudo -u thelounge thelounge -v)" ]]; then

```

Comment:
**sudo -u thelounge** is required since the service/application runs as **thelounge** user.
[Documentation](https://thelounge.chat/docs/install-and-upgrade#debian-and-ubuntu-based-distributions)